tests/fixtures/__init__.py

```diff
@@ -229,6 +237,10 @@ def stream_generate(request):
         token.text = "moose"
         token.logprob = 0.2
         fake_stream.tokens = [token]
+        input_tokens = mock.Mock()
+        input_tokens.text = "moose"
+        input_tokens.logprob = 0.2
+        fake_stream.input_tokens = [input_tokens]
```

Comment:
With the update of these text fixtures, could we add some assertions to the module tests that the new fields are passed along when expected (likely in the `validate_..` functions below in this file)?